bitnami/Makefile

```diff
@@ -0,0 +1,10 @@
+BITNAMI_KAFKA_RELEASE=21.4.2
```

Comment:
Any reason to not move to 22 yet?

Reply:
I'll include the upgrade

Reply:
Upgrade included => only the helm version annotation was updated.